Tests/SwiftDocCTests/Indexing/NavigatorIndexTests.swift

```diff
@@ -419,7 +419,7 @@ Root
             
             let renderIndex = try RenderIndex.fromURL(targetURL.appendingPathComponent("index.json"))
             XCTAssertEqual(renderIndex.interfaceLanguages.keys.count, 1)
-            XCTAssertEqual(renderIndex.interfaceLanguages["swift"]?.count, 27)
+            XCTAssertEqual(renderIndex.interfaceLanguages["swift"]?.count, 29)
```

Comment:
Now that the UncuratedClass is automatically curated Unser SideKit, there are two new items: one for the Classes header and one for the UncuratedClass symbol.

Reply:
What was the reason it wasn't curated before?

Reply:
The [inconsistent path component here](https://github.com/apple/swift-docc/pull/306/files#diff-716310fac3efd50c8b6ec91ef50e428d2f332239c8088c36c8ca97a4c2c8084cL80).

Despite its explicit name it's not clear what it's testing. If it's meant to test handling of inaccurate path data I would still think that we'd want the symbol to end up in the navigator hierarchy of the module that defined the symbol. Also, the `angle` property doesn't include the "NonExistent" path component. I would expect a property to share the same path component prefix as the symbols that it's a member of.